src/doc/rustc/src/codegen-options/index.md

```diff
@@ -382,6 +382,29 @@ the linker.
 
 The default is `yes` if not specified.
 
+## embed-bitcode
+
+This flag controls whether or not the compiler embeds LLVM bitcode into
+generated object files. It takes one of the following values:
+
+* `y`, `yes`, `on`, or no value: Enable bitcode embedding (default).
+* `n`, `no`, or `off`: Disable bitcode embedding.
+
+LLVM bitcode embedding is only needed when link-time optimization (LTO) is
```

Comment:
Maybe worth adding some explicit comment like: Note that the default is yes for backwards compatibility, but we recommend using 'no' unless you are using LTO.

It's kind of obvious from the text here but it requires just a bit of reading between the lines.

Reply:
Good suggestion. I have updated the docs to make this more explicit.